torch_frame/data/multi_nested_tensor.py

```diff
@@ -31,15 +33,34 @@ class MultiNestedTensor(_MultiTensor):
         ...    [torch.tensor([4]), torch.tensor([5, 6, 7])],
         ...    [torch.tensor([8, 9]), torch.tensor([10])],
         ... ]
-        >>> out = MultiNestedTensor.from_tensor_mat(tensor_mat)
-        >>> out.size(0)
-        3
-        >>> out.size(1)
-        2
-        >>> out.size(2)
-        Traceback (most recent call last):
-        File "<stdin>", line 1, in <module>
-        ValueError: MultiNestedTensor does not have a fixed length on the third dimension.  # noqa
+        >>> mnt = MultiNestedTensor.from_tensor_mat(tensor_mat)
+        >>> mnt
+        MultiNestedTensor(num_rows=3, num_cols=2, device='cpu')
+        >>> mnt.values
+        tensor([ 1,  2,  3,  4,  5,  6,  7,  8,  9, 10])
+        >>> mnt.offset
+        tensor([ 0,  2,  3,  4,  7,  9, 10])
+        >>> mnt[0, 0]
+        torch.tensor([1, 2])
+        >>> mnt[1, 1]
+        tensor([5, 6, 7])
+        >>> mnt[0] # Row integer indexing
+        MultiNestedTensor(num_rows=1, num_cols=2, device='cpu')
+        >>> mnt[:, 0] # Column integer indexing
+        MultiNestedTensor(num_rows=3, num_cols=1, device='cpu')
+        >>> mnt[:2] # Row integer slicing
+        MultiNestedTensor(num_rows=2, num_cols=2, device='cpu')
+        >>> mnt[[2, 1, 2, 0]] # Row list indexing
+        MultiNestedTensor(num_rows=4, num_cols=2, device='cpu')
+        >>> mnt.to_dense(fill_value = -1) # Map to a dense matrix via padding
+        tensor([[[ 1,  2, -1],
+                [ 3, -1, -1]],
+
+                [[ 4, -1, -1],
+                [ 5,  6,  7]],
+
+                [[ 8,  9, -1],
+                [10, -1, -1]]])
```

Comment:
Looks weird in the doc page here: https://pytorch-frame--278.org.readthedocs.build/en/278/generated/torch_frame.data.MultiNestedTensor.html#torch_frame.data.MultiNestedTensor

Maybe remove these two empty lines will fix it?
```suggestion
        tensor([[[ 1,  2, -1],
                [ 3, -1, -1]],
                [[ 4, -1, -1],
                [ 5,  6,  7]],
                [[ 8,  9, -1],
                [10, -1, -1]]])
```

Reply:
Good catch! Fixed.